.changeset/purple-weeks-cross.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hono/typebox-validator': minor
```

Comment:
This is a fix change. So you should replace `minor` with `patch`.

Reply:
Ah I see sorry, have corrected it.

Reply:
Thank you!